packages/gatsby-recipes/src/providers/gatsby/plugin.test.js

```diff
@@ -1,24 +1,49 @@
 const fs = require(`fs-extra`)
 const path = require(`path`)
+const tmp = require(`tmp-promise`)
 
 const plugin = require(`./plugin`)
 const { addPluginToConfig, getPluginsFromConfig } = require(`./plugin`)
 const resourceTestHelper = require(`../resource-test-helper`)
 
-const root = path.join(__dirname, `./fixtures/gatsby-starter-blog`)
-const helloWorldRoot = path.join(
+const STARTER_BLOG_FIXTURE = path.join(
+  __dirname,
+  `./fixtures/gatsby-starter-blog`
+)
+const HELLO_WORLD_FIXTURE = path.join(
   __dirname,
   `./fixtures/gatsby-starter-hello-world`
 )
 const name = `gatsby-plugin-foo`
-const configPath = path.join(root, `gatsby-config.js`)
 
 describe(`gatsby-plugin resource`, () => {
+  let tmpDir
```

Comment:
What problems was this causing?

Reply:
Test run modified fixtures without restoring them.

Reply:
Yeah, I think this is the result of prettier resolving a different config (or none at all) so writing back out the config after the last destroy results in a diff.